providers/summary/scancode.js

```diff
@@ -82,7 +83,7 @@ class ScanCodeSummarizer {
 
   _toExpression(licenses) {
     if (!licenses) return null
-    const list = setToArray(licenses)
+    const list = setToArray(licenses).map(correct)
```

Comment:
Similarly here, turn off upgrades? or is that the default?

Reply:
Ahh. Duh. Thanks for finding